net-misc/seafile-client/seafile-client-8.0.6.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+RELEASE_COMMIT="1fb9ddd71fbf6f0252509aced527be459e240366"
+
+inherit xdg cmake
+
+DESCRIPTION="Seafile desktop client"
+HOMEPAGE="https://www.seafile.com/ https://github.com/haiwen/seafile-client/"
+SRC_URI="https://github.com/haiwen/${PN}/archive/${RELEASE_COMMIT}.tar.gz -> ${P}.tar.gz"
```

Comment:
Same as above, since the upstream are still creating tags for each release, you can use `${PV}` here.